sample/dump.c

```diff
@@ -72,12 +83,56 @@ void dumpTree(void* ctx, rmtSampleTree* sample_tree)
 {
     rmtSample* root = rmt_SampleTreeGetRootSample(sample_tree);
     const char* thread_name = rmt_SampleTreeGetThreadName(sample_tree);
+    if (strcmp("Remotery", thread_name) == 0)
+    {
+        return; // to minimize the verbosity in this example
+    }
 
     printf("// ********************   DUMP TREE: %s   ************************\n", thread_name);
 
     printTree(root, 0);
 }
 
+void printProperty(rmtProperty* property, int indent)
+{
+    rmtPropertyIterator iter;
+
+    const char* name = rmt_PropertyGetName(property);
+    rmtPropertyType type = rmt_PropertyGetType(property);
+    rmtPropertyValue value = rmt_PropertyGetValue(property);
```

Comment:
The iterator use case:
Should we get the union value, or use specific getters for each type?

Reply:
Make `rmtPropertyValue` public and let the user access that based on type.